apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -17,7 +20,18 @@ const AnimationPickerBody = React.createClass({
     onUploadClick: React.PropTypes.func.isRequired
   },
 
+  getInitialState() {
+    return {
+      searchQuery: ''
+    };
+  },
+
+  onSearchQueryChange(value) {
+    this.setState({searchQuery: value});
+  },
+
   render() {
+    var pageOfResults = searchAnimations(this.state.searchQuery);
```

Comment:
Checking my understanding — `page` here just means the result set is limited, not that there's pagination involved?